### ISSUE TITLE
File upload issue

### ISSUE THREAD

**Issue by inflameous48** [ezl/dailyidea.com#445]
There seems to currently be an issue with attachments, screenshot below:

![image](https://user-images.githubusercontent.com/50846999/85863438-d18acc80-b7cb-11ea-9c28-c2824527499c.png)

**Comment by vedmant** [comment]
@inflameous48 What's exact steps to reproduce this error? I just tried to upload images and it works ok for me, I don't see the code you highlighted. 

@ezl I see there are few another issues: 

1. Content editor height: it's not taking the space it's supposed to take: https://prnt.sc/t8yihd I think this also should be fixed.
1. When I deleted image and then pressed CTL-Z I see "An error occurred".
1. Dragndrop issue, when image dropped on top of uploaded image, it opens image in the browser and loses all typed data.
1. No way to cancel image uploading? I just uploaded list of images and some are large and take quite some time to upload, wanted to cancel uploading but there is no way to do this.

**Comment by ezl** [comment]
1. content editor height -- lets hold off on this for now. @alisonuxdesign has new screens coming up, so I think it makes sense to wait for those before making aesthetic / UI bug fixes.
2. delete image -- can you ticket this please and go ahead and assign it to yourself for execution now
3. dragndrop issue -- please ticket and self assign for completion now
4. cancel image upload -- ticket this, but do not assign it yet. low priority IMO, so not focused on fixing yet.

**Comment by ezl** [comment]
@inflameous48 can you create steps to reproduce this error?

**Comment by inflameous48** [comment]
@vedmant @ezl looks like this issue has been fixed, it wasn't originally discovered by me but from what I saw there was a deployment shortly after and that might've fixed that. Another thing I've noticed is the name of the file and its file size displaying twice, I was wondering if that is intended?

![image](https://user-images.githubusercontent.com/50846999/86688973-3f769580-c00f-11ea-9578-35567ced6446.png)

**Comment by vedmant** [comment]
@inflameous48 How do I reproduce this? I just tested uploading image and it shows filename with size only once.

**Comment by ezl** [comment]
can't reproduce. closing. if it comes back we'll deal with it then.
